MachObfuscator/SymbolMangling/RealWordsMangler/RealWordsMangler.swift

```diff
@@ -70,7 +70,7 @@ private extension ObfuscationSymbols {
     func classManglingMap(sentenceGenerator: SentenceGenerator) -> [(String, String)] {
         let mangledClassesBlacklist = (Array(blacklist.classes) + Array(whitelist.classes)).uniq
         let classManglingEntryProvider: (String) -> (String, String)? = { className in
-            while let randomClassName = sentenceGenerator.getUniqueSentence(length: className.count)?.capitalizedOnFirstLetter {
+            while let randomClassName = sentenceGenerator.getUniqueSentence(length: className.utf8.count)?.capitalizedOnFirstLetter {
```

Comment:
Could you please add some unit tests for those changes?

Reply:
tests added